### PR TITLE
Removes hippie fusion

### PR DESCRIPTION
**Pull request by ghost**
Assmos added real fusion

:cl: Thefastfoodguy
del: Removes hippie fusion
/:cl:

### CODE REVIEW

**Comment by KayeArray** [comment]
!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by ghost** [comment]
@banthebantz yes but ours is unrealistic

**Review by ghost — DISMISSED** [review]

**Comment by GuyonBroadway** [comment]
Fack yoo and yer realism.

File: hippiestation/code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -1,55 +0,0 @@
-#define PLASMA_BINDING_ENERGY 3000000
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/atmospherics/gasmixtures/reactions.dm:1]: YOU FORGOT TO UNSTAGE THE CHANGE IN HIPPIESTATION.DME FUCCBOI!

**Review by GuyonBroadway — DISMISSED** [review]

**Comment by KayeArray** [comment]
ours is pretty and magical and colourful

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**